gnoi_reset/gnoi_reset.go

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Binary implements a gNOI Reset.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"google.golang.org/grpc"
+
+	log "github.com/golang/glog"
+
+	"github.com/google/gnxi/gnoi/reset"
+	"github.com/google/gnxi/utils/credentials"
+)
+
+var (
+	targetAddr = flag.String("target_addr", "", "The target address in the format of host:port")
+	targetName = flag.String("target_name", "", "The target name used to verify the hostname returned by TLS handshake")
+	rollbackOs = flag.Bool("rollback_os", false, "Target must revert to factory OS")
+	zeroFill   = flag.Bool("zero_fill", false, "Target must overwrite persistent storage with zeroes")
+	timeOut    = flag.Duration("time_out", 10*time.Second, "Timeout for ResetTarget operation, 10 seconds by default")
```

Comment:
nit: Best to give more descriptive name in case other timeouts are added later. Maybe: "reset_target_timeout".

nit: I would also include an example of what units are required. flag.Duration takes anything parsable by flag.ParseDuration so for example the user could use:

"Timeout for ResetTarget operation, 10 seconds by default e.g. 10s"